src/main/kotlin/blackjack/domain/player/Player.kt

```diff
@@ -0,0 +1,42 @@
+package blackjack.domain.player
+
+import blackjack.domain.Result
+import blackjack.domain.card.Card
+import blackjack.domain.card.Cards
+
+abstract class Player(
+    val name: String
+) {
+
+    val cards: Cards = Cards()
```

Comment:
```suggestion
abstract class Player(
    val name: String,
    val cards: Cards = Cards()
) {
```
생성자에 주입받을수 있는구조로 작성하면 어떤 장점이 있을까요?
원하는 카드를 가진 player를 테스트하기 좋지않을까요?